docs/upgrade.md

```diff
@@ -6,202 +6,133 @@ sidebar_label: Upgrade
-storageclass "openebs-percona" deleted
-storageclass "openebs-jupyter" deleted
-storageclass "openebs-mongodb" deleted
-storageclass "openebs-cassandra" deleted
-storageclass "openebs-redis" deleted
-storageclass "openebs-kafka" deleted
-storageclass "openebs-zk" deleted
-storageclass "openebs-es-data-sc" deleted
-```
+### Upgrade volume policies in existing storage classes
 
-Wait for the objects to be deleted. You can verify the same using the following command.
+Use the following command to upgrade the storage classes volume policies. Move from parameters to annotations. This script will only update the following volume policies:
 
-```
-kubectl get deploy
-```
+- `openebs.io/replica-count`
+- `openebs.io/storage-pool`
+- `openebs.io/volume-monitor`
 
-You can now install the 0.6 operator using following command.
+The remaining policies will fallback to their default values. 
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-operator.yaml
+./upgrade_sc.sh
 ```
 
-The output must be similar to the following.
+Alternatively, you can skip this step and re-apply your StorageClasses as per the 0.7.0 volume policy specification. 
 
-```
-namespace "openebs" created
-serviceaccount "openebs-maya-operator" created
-clusterrole "openebs-maya-operator" created
-clusterrolebinding "openebs-maya-operator" created
-deployment "maya-apiserver" created
-service "maya-apiserver-service" created
-deployment "openebs-provisioner" created
-deployment "openebs-snapshot-operator" created
-customresourcedefinition "storagepoolclaims.openebs.io" created
-customresourcedefinition "storagepools.openebs.io" created
-storageclass "openebs-standard" created
-storageclass "openebs-snapshot-promoter" created
-customresourcedefinition "volumepolicies.openebs.io" created
-```
+**Note: StorageClasses have to updated prior to provisioning any new volumes with 0.7.0.**
 
-You must deploy the 0.6 storage classes using the following command.
+### Upgrading OpenEBS Operator CRDs and Deployments
 
-```
-kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-storageclasses.yaml
-```
+The upgrade steps vary depending on the way OpenEBS was installed, select one of the following:
 
-The output must be similar to the following.
+#### Install/Upgrade using kubectl (using openebs-operator.yaml )
 
-```
-storageclass "openebs-standalone" created
-storageclass "openebs-percona" created
-storageclass "openebs-jupyter" created
-storageclass "openebs-mongodb" created
-storageclass "openebs-cassandra" created
-storageclass "openebs-redis" created
-storageclass "openebs-kafka" created
-storageclass "openebs-zk" created
-storageclass "openebs-es-data-sc" created
-```
+**The sample steps below will work if you have installed openebs without modifying the default values in openebs-operator.yaml. If you have customized it for your cluster, you will have to download the 0.7.0 openebs-operator.yaml and cutomize it again**
```

Comment:
... values in the *openebs-operator.yaml* file.
you will have to = you must download
0.7.0 openebs-operator.yaml ile and customize it again.**